mlflow/paddle/__init__.py

```diff
@@ -222,7 +228,7 @@ def forward(self, inputs):
     mlflow_model.save(os.path.join(path, MLMODEL_FILE_NAME))
 
 
-def load_model(model_uri):
+def load_model(model_uri, model=None, skip_mismatch=False, reset_optimizer=False):
```

Comment:
```suggestion
def load_model(model_uri, model=None, **kwargs):
```

Can we use `kwargs` here and pass it to `paddle.jit.load` or `model.load` in the function?

```python
if model is None:
    paddle.jit.load(..., **kwargs)
else:
    model.load(..., **kwrags)
```